test/manager/pip_setup/__snapshots__/extract.spec.ts.snap

```diff
@@ -128,6 +128,134 @@ Object {
 }
 `;
 
+exports[`lib/manager/pip_setup/extract extractPackageFile() should return null for invalid file 1`] = `
```

Comment:
Revert all changes to this file

Reply:
This change can also occur if python mock lib is missing. Included since python 3.3

`apt-get install -y python-mock`
or
`pip install mock`